### ISSUE TITLE
<ix-tabs> The tabs sliding window position is incorrect if tab selected by JS code

### ISSUE THREAD

**Issue by chencc1988** [siemens/ix#1013]
### What happened?

When the tab list exceeds the width of the container, the sliding window of the tabs does not move to the selected tab position if the tab's selected state is controlled by javascript.

In the reproduction demo code, the tabs are force to lock to tab1. However if you click the tab5, the tabs position will stay on tabs 5 but not moves back to the tab1. 

The real case is if the page is refreshing and the tab that needs to be displayed is the 5th tab (in my application this is determined by angular's routing parameter). The user will not be aware of that the 5th tab is selected. Instead, they think that no tab is selected

![Animation](https://github.com/siemens/ix/assets/10339536/7e46ba41-cb13-4f7d-af5c-2e79ffa1a850)

### What type of frontend frameware are you seeing the problem on?

Angular

### Which version of iX do you use?

v2.0.4

### Code to produce this issue.

```shell
https://stackblitz.com/edit/1tqmdv?file=src%2Fapp%2Ftabs.ts
```

**Comment by matthiashader** [comment]
Internal ref [IX-866]

**Comment by thiemodaubner** [comment]
+1 - Facing the same issue

**Comment by nuke-ellington** [comment]
Fixed via #1390
